.ci.yaml

```diff
@@ -243,8 +243,6 @@ targets:
 
   - name: Linux packages_autoroller
     presubmit: false
-    # TODO(fujino): https://github.com/flutter/flutter/issues/129744
-    bringup: true
```

Comment:
I would prefer to leave this bringup until I verify this works--however, as the staging pool does not have access to the cloudkms key, it must run in prod to verify the fix.

Reply:
I am the framework gardener this week, however, so I will fix the tree if I break it.